public/media-hunt/commands/movies/movies.js

```diff
@@ -1,8 +1,19 @@
 const { getMoviesDiscover } = require("../../http/movie");
+const { handleDatabase } = require("../../database/database");
 
 const run = async () => {
-    const data = await getMoviesDiscover();
-    console.log(data);
-  };
+  {
+    let data;
+    if (handleDatabase.checkIfExists()) {
+      data = handleDatabase.getDataByKey();
+    } else {
+      const response = await getMoviesDiscover();
+      data = handleDatabase.storeDataByKey(response);
+    }
+    console.log("data here", data);
+  }
+  const data = await getMoviesDiscover();
+  console.log(data);

```

Comment:
We said we won't be caching this response, since this is our entry point in app exploration to find movies we wanna get details for. And this changes every now and then to show popular movies.

Reply:
done